### ISSUE TITLE
Release DXF examples in a separate tarball, rpm, ..., etc.

### ISSUE THREAD

**Issue by bert** [bert/libdxf#41]

**Comment by paolo-caroni** [comment]
Good idea, but imho the DXF examples are useful for developer, so we can take it with developer documentation (doxygen) and DXF change over years documentation (that written by me).
however I agree with any choice you think good.
Cheers,
Paolo
